### ISSUE TITLE
Improve plotting script

### ISSUE THREAD

**Issue by ilyes319** [ACEsuit/mace#524]
**Feature request**
The plotting script currently lacks functionality: 
- Plotting cleaner different training loss/ validation loss 
- Add labels to the plot (swa, patience).
